charts/otel-cloud-stack/kubelet_scrape_configs.yaml

```diff
@@ -0,0 +1,244 @@
+# This is used for scrape kubelet
```

Comment:
@jaronoff97 can we replace the vanilla prom config with podMonitor/serviceMonitor with targetAllocator ?

Reply:
this would also require us to run a target allocator for the daemonset collector which feels a bit superfluous. That being said, this config is being used by a target allocator already... i'll discuss with the SIG to get their thoughts